README.md

```diff
@@ -47,7 +47,7 @@ Go to Settings -> Themes, and select a Catppuccin theme from the dropdown.
 3. Paste into theme box, and click install.
 
 ### Forks
-This theme works on Misskey forks, such as [Calckey](https://codeberg.org/calckey/calckey) and [Foundkey](https://akkoma.dev/FoundKeyGang/FoundKey).
+This theme works on Misskey forks, such as [Firefish](https://codeberg.org/firefish/firefish) and [Foundkey](https://akkoma.dev/FoundKeyGang/FoundKey).
```

Comment:
The codeberg README says the following:

![image](https://github.com/catppuccin/misskey/assets/58985301/aba157e1-2f80-44fb-9138-011b9666cf6d)

We should update the link of `Firefish` to point towards `https://git.joinfirefish.org/firefish/firefish` as suggested below:

```suggestion
This theme works on Misskey forks, such as [Firefish](https://git.joinfirefish.org/firefish/firefish) and [Foundkey](https://akkoma.dev/FoundKeyGang/FoundKey).
```